x/wasm/keeper/keeper_extension.go

```diff
@@ -0,0 +1,66 @@
+package keeper
+
+import (
+	sdk "github.com/line/lbm-sdk/types"
+	sdkerrors "github.com/line/lbm-sdk/types/errors"
+	wasmtypes "github.com/line/lbm-sdk/x/wasm/types"
+)
+
+func (k Keeper) IsInactiveContract(ctx sdk.Context, contractAddress sdk.AccAddress) bool {
+	store := ctx.KVStore(k.storeKey)
+	return store.Has(wasmtypes.GetInactiveContractKey(contractAddress))
+}
+
+func (k Keeper) IterateInactiveContracts(ctx sdk.Context, fn func(contractAddress sdk.AccAddress) (stop bool)) {
+	store := ctx.KVStore(k.storeKey)
+	prefix := wasmtypes.InactiveContractPrefix
+	iterator := sdk.KVStorePrefixIterator(store, prefix)
+	defer iterator.Close()
+
+	for ; iterator.Valid(); iterator.Next() {
+		contractAddress := sdk.AccAddress(iterator.Value())
+		if stop := fn(contractAddress); stop {
+			break
+		}
+	}
+}
+
+func (k Keeper) addInactiveContract(ctx sdk.Context, contractAddress sdk.AccAddress) {
+	store := ctx.KVStore(k.storeKey)
+	key := wasmtypes.GetInactiveContractKey(contractAddress)
+
+	store.Set(key, contractAddress)
```

Comment:
Is this value (`contractAddress`) needed?

Reply:
Yes, this value is used when the `IterateInactiveContracts` returns a list of inactive contracts.